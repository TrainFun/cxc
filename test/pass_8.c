int main() {
  for (double x = 0.123456; x < 10.; x = x + 2.) {
    write x;
  }
}
