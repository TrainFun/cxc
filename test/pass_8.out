0.123456
2.123456
4.123456
6.123456
8.123456
